x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/empty_state.tsx

```diff
@@ -18,7 +18,7 @@ export const EmptyState = memo<{
 }>(({ onAdd, isAddDisabled = false, backComponent }) => {
   return (
     <ManagementEmptyStateWrapper>
-      <EuiEmptyPrompt
+      <EuiPageTemplate.EmptyPrompt
```

Comment:
Q: Should be okay to use the individual `EmptyPrompt` component here? No?

Reply:
see this comment: https://github.com/elastic/kibana/pull/162531#discussion_r1307861905